source/includes/administration/_environments.md

```diff
@@ -196,25 +253,69 @@ curl -X POST "https://cloudmc_endpoint/api/v1/environments/f9dea588-d7ab-4f42-b6
 
 ```json
 {
-  "name": "skynet-beta",
-  "description": "Environment for the Skynet project",
-  "roles": [{
-    "id": "f9dea588-d7ab-4f42-b6e6-4b85f273f3db",
-    "users": [{
-      "id": "07e02355-d05b-47cf-860d-f69cf0432276"
-    }]
-  }]
+  "name": "skynet-beta-v2",
+  "description": "Environment for the Skynet project (updated)",
 }
 ```
 
 
 Optional | &nbsp;
 -------- | -----------
 `name`<br/>*string* | The updated name of the environment. Should be unique in the environment and only contain lower case characters, numbers, dashes and underscores.
-`description`<br/>*string* | The updated description of the environment.
-`membership`<br/>*string* | Type of membership of the environment. ALL_ORG_USERS will add every user in the organization to this environment with the default role. MANY_USERS will allow you to choose the users you want in the environment and assigned them specific roles. *Defaults to MANY_USERS*.
-`roles`<br/>*Array[[Role](#administration-roles)]* | Update the users roles in the environment. Also, can also update the default role.<br/>*required*: `name`, `users.id`<br/>*optional*: `isDefault`
+`description`<br/>*string* | The updated description of the environment
+`membership`<br/>*string* | Type of membership of the environment. ALL_ORG_USERS will add every user in the organization to this environment with the default role. MANY_USERS will allow you to  choose the users you want in the environment and assigned them specific roles. *Defaults to MANY_USERS*
 
+Response | &nbsp;
+---------- | -----------
+`taskId`<br/>*UUID* | The id of the task
+`taskStatus`<br/>*string* | The status of the task
+`data`<br/>*[Environment](#administration-environments)* | The information about the updated environment
+
+```shell
+# Response body example
+```
+```json
+{
+  "data": {
+    "metadata": {
+      "mode": "project"
+    },
+    "defaultRole": {
+      "creationDate": "2020-01-02T16:04:27.000Z",
+      "version": 1,
+      "isSystem": false,
+      "isDefault": false,
+      "deleted": false,
+      "name": "Owner",
+      "alias": "owner",
+      "id": "b7c48102-350e-40b9-9a3e-b7a0b3cfa535",
+      "isFixed": true,
+      "defaultScope": "ENV"
+    },
+    "description": "Environment for the Skynet project (updated)",
+    "membership": "MANY_USERS",
+    "creationDate": "2020-02-27T16:23:20.000Z",
+    "deleted": false,
+    "organization": {
+      "name": "System",
+      "id": "f6ece376-de9e-45d5-8719-82a279664ea9",
+      "entryPoint": "system"
+    },
+    "name": "skynet-beta-v2",
+    "id": "d77de7ad-274e-4ec9-8776-8d748b0671fd",
+    "allowExternalMembers": false,
+    "state": "PROVISIONED",
+    "serviceConnection": {
+      "serviceCode": "acs-dev-1",
+      "name": "acs-dev-1",
+      "id": "8901494c-01ee-4d6b-bd12-cd5347127039",
+      "type": "cloudstack"
+    }
+  },
+  "taskId": "105cddf8-4507-4131-87eb-fe383bbd882b",
+  "taskStatus": "PENDING"
```

Comment:
These fields should go above `data`. Also, I think you only get the environment returned when the task is `SUCCESS`